### ISSUE TITLE
Problems with adding a keyboard

### ISSUE THREAD

**Issue by perrottt** [fluddokt/opsu#186]
**Version:** 0.15.0b
**OS:** Linux (armv7l)
**JRE:** The Android Project 0
**OpenGL version:** OpenGL ES 3.0 V@100.0 AU@  (GIT@I716aebe4f9) (Qualcomm)
BOARD: MSM8916
FINGERPRINT: samsung/gtelwifiue/gtelwifiue:5.1.1/LMY47X/T560NUUEU1AOH6:user/release-keys
HOST: SWHE0718
MODEL: SM-T560NU
INCREMENTAL: T560NUUEU1AOH6
RELEASE: 5.1.1

**Error:** `RenderError`
**Stack trace:**
```
java.lang.NullPointerException: Attempt to invoke virtual method 'int java.lang.String.length()' on a null object reference
	at fluddokt.opsu.fake.DynamicFreeTypeFont.getWidth(DynamicFreeTypeFont.java:295)
	at fluddokt.opsu.fake.Font.getWidth(Font.java:83)
	at itdelatrisu.opsu.options.OptionsOverlay.renderGenericOption(OptionsOverlay.java:749)
	at itdelatrisu.opsu.options.OptionsOverlay.renderOption(OptionsOverlay.java:642)
	at itdelatrisu.opsu.options.OptionsOverlay.renderOptions(OptionsOverlay.java:582)
	at itdelatrisu.opsu.options.OptionsOverlay.render(OptionsOverlay.java:478)
	at itdelatrisu.opsu.states.SongMenu.render(SongMenu.java:828)
	at fluddokt.opsu.fake.StateBasedGame.render(StateBasedGame.java:134)
	at fluddokt.opsu.fake.GameOpsu.render(GameOpsu.java:129)
	at com.badlogic.gdx.backends.android.AndroidGraphics.onDrawFrame(AndroidGraphics.java:422)
	at android.opengl.GLSurfaceView$GLThread.guardedRun(GLSurfaceView.java:1531)
	at android.opengl.GLSurfaceView$GLThread.run(GLSurfaceView.java:1248)
```

**Comment by perrottt** [comment]
I tried to connect a USB mechanical numpad to my Samsung Tab E using a USB female to micro USB male OTG cable. Every time I try to assign a key for left and right in settings, it comes out with this problem.

**Comment by fluddokt** [comment]
Probably fixed in 0.16.0a
